bower.json

```diff
@@ -20,11 +20,11 @@
     "tests"
   ],
   "dependencies": {
-    "angular": "~1.2",
-    "angular-route": "~1.2",
-    "lodash": "~2.4"
+    "angular": "1.2 - 1.5",
+    "angular-route": "1.2 - 1.5",
+    "lodash": "2.4 - 4"
```

Comment:
@henrahmagix  looking at the docs the descriptions are the same

4.2
https://lodash.com/docs#memoize

2.4
https://github.com/lodash/lodash/blob/2.4.2/doc/README.md#_memoizefunc-resolver